t/plugin/proxy-rewrite2.t

```diff
@@ -202,7 +202,7 @@ GET /echo
 X-Forwarded-Proto: http
 X-Forwarded-Proto: grpc
 --- response_headers
-X-Forwarded-Proto: http
+X-Forwarded-Proto: http, grpc
```

Comment:
Why do you add a `grpc` here?

Reply:
The response header has `http, grpc`. I don't know why it looks like this.

Reply:
It's werid, could you dive deep into this?

Reply:
OK, I will take a look.